conf/uvek.conf.part

```diff
@@ -1470,6 +1470,42 @@ source src_ch_bfe_biogasanlagen : def_searchable_features
          from bfe.biogasanlagen
  }
 
+source src_ch_bfe_erneuerbarheizen : def_searchable_features
+ {
+     sql_db = uvek_dev
+     sql_query = \
+         SELECT bgdi_id as id \
```

Comment:
as already mentioned in the chsdi review, please use the customer id xtf_id if they provide one.